app/rpc/tests/rpc_test.go

```diff
@@ -586,9 +586,8 @@ func TestEth_GetTransactionLogs(t *testing.T) {
 	require.True(t, transactionLogs[0].Topics[1].Big().Cmp(big.NewInt(1024)) == 0)
 
 	// inexistent tx hash
-	rpcRes, err := CallWithError("eth_getTransactionLogs", []interface{}{inexistentHash})
-	require.NoError(t, err)
-	assertNullFromJSONResponse(t, rpcRes.Result)
+	_, err := CallWithError("eth_getTransactionLogs", []interface{}{inexistentHash})
```

Comment:
why change here?

Reply:
just make the rpc ut pass through, by the way